ckanext/datavic_odp_theme/plugin.py

```diff
@@ -52,6 +53,12 @@ def get_auth_functions(self):
     
     # IBlueprint
     def get_blueprint(self):
+        # Check feature preview is enabled or not
+        # If enabled add the redirect view for read pkg
+        preview_redirect_enabled = bool(config.get(
```

Comment:
It is still not ideal. `ckan.dataset.preview_redirect = false` in the config file produces the following expression here: `bool('false')`. And, as any non-empty string, it turns into `True`. Use `tk.asbool` instead